audiorecorder/src/main/java/org/odk/collect/audiorecorder/Config.kt

```diff
@@ -0,0 +1,67 @@
+package org.odk.collect.audiorecorder
+
+import android.app.Activity
+import android.app.Application
+import android.media.MediaRecorder
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.odk.collect.audiorecorder.recorder.MediaRecorderRecorder
+import org.odk.collect.audiorecorder.recorder.RealMediaRecorderWrapper
+import org.odk.collect.audiorecorder.recorder.Recorder
+import org.odk.collect.audiorecorder.recording.AudioRecorderActivity
+
+private var _component: AudioRecorderDependencyComponent? = null
```

Comment:
This file provides a pretty minimal (for Dagger) DI setup. We could come up with something more complex to attach the component to the `Application` object which would mean we wouldn't run into static state problems (see the code comment below) but I don't think we need to go there yet.